### PR TITLE
issue 2908 fix

### PR DESCRIPTION
**Pull request by PrakashDurlabhji**

### CODE REVIEW

**Comment by gurmeetb** [comment]
@PrakashDurlabhji please make sure PR passes before marking `ready for review`.

**Comment by PrakashDurlabhji** [comment]
@gurmeetb yes doing all

**Comment by gurmeetb** [comment]
Thanks @PrakashDurlabhji

**Comment by vl-ti** [comment]
@PrakashDurlabhji 
Please check issue description. It is required to navigate across the info links (green icons with question marks) and when it is selected the popup info message should be visible.
Current fix navigates between labels.

**Comment by PrakashDurlabhji** [comment]
@vl-ti PR updated

**Comment by vl-ti** [comment]
@PrakashDurlabhji Hi! Some issues are still there:
* There is no need to focus on labels (h4, h3 html tags)
* When info link is selected, the popup message must be visible. The same behaviour as on mouse hover over the icon.

**Comment by PrakashDurlabhji** [comment]
@vl-ti Pr updated

**Comment by sexta13** [comment]
@PrakashDurlabhji Still not working. I can access the ? points, but no data is shown.
"Expected Result
Should able to access the data on the info popup by tab key"

image

**Comment by veshu** [comment]
@PrakashDurlabhji Please check this as soon as early.

File: src/shared/components/challenge-detail/Specification/SideBar/index.jsx [hunk 1]
```diff
@@ -108,8 +108,8 @@ Final Review:
             </h4>
             <span styleName="link-like-paragraph tooltip-container">
               {reviewTypeTitle}
-              <Tooltip content={reviewTip}>
-                <div styleName="tctooltip">
+              <Tooltip name="finalReview" content={reviewTip}>
+                <div role="button" tabIndex={0} styleName="tctooltip" aria-describedby="finalReview">
```
[the remark this answers is not in the document]
Comment (vl-ti) [reply src/shared/components/challenge-detail/Specification/SideBar/index.jsx]: Such way of defining number constants  in jsx is used all over the project, so I believe it make sense to keep it in order to have same code style

**Comment by vl-ti** [comment]
@PrakashDurlabhji
What was the reason of adding additional prop “name”?

File: src/shared/components/challenge-detail/Specification/SideBar/index.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (sexta13) [reply src/shared/components/challenge-detail/Specification/SideBar/index.jsx]: I understand that point, if you had a constant all over the project called for instance: defaultTabIndex = 0, and then use it in tabIndex={defaultTabIndex}.
This way, you are just adding proccessing to something that you already know it's static. Anyway, if you are ok with that, no problem for me :)

**Pull request closed without merging**